README.md

```diff
@@ -33,11 +33,11 @@ In the rapidly evolving world of blockchain technology, effective monitoring is
 
 ### Installation
 
-View the [Installation](https://docs.openzeppelin.com/monitor/0.1.0/#getting_started) documentation for detailed information. For a quicker introduction, check out the [Quickstart](https://docs.openzeppelin.com/monitor/0.1.0/quickstart) guide.
+View the "Get Started" details in [documentation](https://docs.openzeppelin.com/monitor/) for detailed information. For a quicker introduction, check out the "Quick Start" guide in documentation.
```

Comment:
I prefer the current style of links, I have an active [PR](https://github.com/OpenZeppelin/openzeppelin-monitor/pull/119) that removed the versioning from the links already in readme and contributing files.

Reply:
Ok nice